ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host2/ip-dhcp-relay.yml

```diff
@@ -0,0 +1,5 @@
+---
+### IP DHCP Relay ###
+
```

Comment:
This is already covered in host1.

Reply:
it is to improve coverage by not defining other keys